src/main/resources/static/js/book.js

```diff
@@ -0,0 +1,96 @@
+function myFunc1() {
```

Comment:
Better name then myFunc1

Reply:
Let's add a todo to remove it, or name it as a developer function